pkg/vc/command.go

```diff
@@ -83,19 +101,49 @@ func (f *Framework) SignPresentation(presentation []byte, privKey []byte, opts *
 }
 
 // VerifyPresentation verifies a proof in the verifiable presentation.
-func (f *Framework) VerifyPresentation(vp []byte, pubKey []byte, pubKeyType string) error {
-	_, err := verifiable.ParsePresentation(
+// If there is a presentation definition provided, also verifies that the presentation meets the requirements.
+func (f *Framework) VerifyPresentation(vp []byte, pdBz []byte) (*verifiable.Presentation, error) {
+	// verify VP
+	presentation, err := verifiable.ParsePresentation(
 		vp,
-		verifiable.WithPresPublicKeyFetcher(verifiable.SingleKey(pubKey, pubKeyType)),
+		verifiable.WithPresPublicKeyFetcher(f.resolver.PublicKeyFetcher()),
 		verifiable.WithPresJSONLDDocumentLoader(f.loader),
 	)
 	if err != nil {
-		return fmt.Errorf("failed to verify presentation: %w", err)
+		return nil, fmt.Errorf("failed to verify presentation: %w", err)
 	}
-	return nil
+
+	// verify presentation against PD
+	if pdBz != nil {
```

Comment:
If 'pd' is null, it is always judged to be correct. It is not problem?

Reply:
It is not mandatory to have presentation definition for exchanging VP.
I think we also have to cover the case that there is no pd as an sdk.

And it depends on the application.
For our oracle case, presentation definition will be nil when json schema validation is used.

Reply:
Yes. If PD is optional, I think that should be handled by the client using it. 
If so, I think the common module, vc-sdk, could provide two functions.
One is `VerifyPresentation(vc)` without pd, and one is `VerifyPresentationWithPD(vc, pd)` with pd. 
The function that accepts both, like `VerifyPresentation(vc, pd)`, has the potential for the client to think that pd is mandatory.

Actually, the above is just my personal opinion, and I like the code as it is now. :)

Reply:
I see what you're talking about.
Let me think about it.

Reply:
How about using option like `WithPresentationDefinition`?

Reply:
Yes. It looks good.
Maybe we can add more options later.

Reply:
Thank you.
Done in #14.